### PR TITLE
[DONE]pr/up&going/ch3/###types&grammar

### PR DESCRIPTION
**Pull request by Zenfeuer**
Initial commit

**Yes, I promise I've read the [Contributions Guidelines](https://github.com/getify/You-Dont-Know-JS/blob/master/CONTRIBUTING.md)** (please feel free to remove this line).

### CODE REVIEW

**Comment by EstebanMarin** [comment]
Thanks for your time

**Comment by Zenfeuer** [comment]
Done for review @mballeng91 @EstebanMarin

File: up & going/ch3.md [hunk 1]
```diff
@@ -39,19 +39,19 @@ The claims I make regarding delegation versus inheritance come not from a dislik
...
 
-By far, the conventional wisdom is that implicit coercion is a "bad part" of the language and should be avoided at all costs. In fact, some have gone so far as to call it a "flaw" in the design of the language. Indeed, there are tools whose entire job is to do nothing but scan your code and complain if you're doing anything even remotely like coercion.
+En gran medida, la sabiduría convencional es que la coerción implícita es una "mala parte" del lenguaje y debería evitarse a toda costa. De hecho, algunos han ido tan lejos en llamarlo una "falla" en el diseño del lenguaje. En realidad hay herramientas cuyo su único trabajo es no hacer nada más que escanear su código y quejarse si usted está haciendo algo remotamente parecido a la coerción.
 
-But is coercion really so confusing, so bad, so treacherous, that your code is doomed from the start if you use it?
+Pero es realmente la coerción tan confuso, tan malo, tan traicionero, que su código está condenado desde el momento que usted lo usa?
```
Comment (mballeng91) [inline up & going/ch3.md]: Hi Darwing, we can use 'confusa', 'mala', 'traicionera'... Switch adjectives to these forms due you're using 'la'

File: up & going/ch3.md [hunk 2]
```diff
@@ -39,19 +39,19 @@ The claims I make regarding delegation versus inheritance come not from a dislik
...
 
-But I'm not merely suggesting that coercion is sensible and learnable, I'm asserting that coercion is an incredibly useful and totally underestimated tool that *you should be using in your code.* I'm saying that coercion, when used properly, not only works, but makes your code better. All the naysayers and doubters will surely scoff at such a position, but I believe it's one of the main keys to upping your JS game.
+Pero no estoy sólo sugiriendo que la coerción es razonable y aplendible, estoy afirmando que la coerción es una herramienta increíblemente útil y totalmente subestimada que *usted debería usar en su código*. Estoy diciendo que esa coerción, cuando se usa adecuadamente, no sólo funciona, sino hace que su código sea mejor. Seguramente todos los que no están de acuerdo con esto se burlarán de esta posición sobre la coerción, pero creo que es una de las principales claves para aumentar tu juego en JS.
```
Comment (mballeng91) [inline up & going/ch3.md]: We can use 'se puede aprender' instead of 'aprendible' that what you meant on first line

File: up & going/ch3.md (hunk 1 above)
Comment (Zenfeuer) [reply up & going/ch3.md]: Hi @mballeng91, it is done, thanks!

File: up & going/ch3.md (hunk 2 above)
Comment (Zenfeuer) [reply up & going/ch3.md]: Done @mballeng91 , thanks!

**Comment by mballeng91** [comment]
Thanks @Zenfeuer ! :1st_place_medal:

**Pull request merged**